app/processing/settlement/get-supporting-settlements.js

```diff
@@ -0,0 +1,19 @@
+const settlement = require('../../data/models/settlement')
+const getCompletedInvoiceNumbers = require('./get-completed-invoice-numbers')
+const getLastSettlement = require('./get-last-settlement')
+const getSettlementsByInvoiceNumber = require('./get-settlements-by-invoice-number')
+const getPaymentValue = require('./get-payment-value')
+const validateSupportingSettlements = require('./validate-supporting-settlements')
+
+const getSupportingSettlements = async (settlementDate, agreementNumber, marketingYear, transaction) => {
+  const completedInvoiceNumbers = await getCompletedInvoiceNumbers(agreementNumber, marketingYear, transaction)
+  const supportingSettlements = await getSettlementsByInvoiceNumber(settlementDate, completedInvoiceNumbers, transaction)
+  validateSupportingSettlements(completedInvoiceNumbers, supportingSettlements)
+  for (const supportingSettlement of supportingSettlements) {
+    const lastSettlement = await getLastSettlement(supportingSettlement.settlementDate, supportingSettlement.value, supportingSettlement.invoiceNumber, transaction)
+    settlement.paymentValue = getPaymentValue(supportingSettlement.value, lastSettlement?.value)
+  }
+  return supportingSettlements
+}
```

Comment:
Need to get all AP settlements submitted to DAX so we can calculate true payment value.  We also have to work out payment value in same way as first payments.